src/appendices/feedback.rst

```diff
@@ -0,0 +1,82 @@
+Feedback
+========
+
+Welcome LaunchCoder! We're so glad you're here. LaunchCoders help make 
+our curriculum stronger. You're a part of this community as a learner 
+and we want you to contribute to making our classes stronger.
```

Comment:
"making our classes stronger" feels repetitive to "make our curriculum stronger" in the last sentence. I may just use "contribute to our classes".